package.json

```diff
@@ -9,7 +9,8 @@
     "dev": "NODE_ENV=development npm run start",
     "dev:search": "APP_NAME=search npm run dev",
     "dev:disco": "APP_NAME=disco npm run dev",
-    "test": "npm run version-check && npm run lint",
+    "test": "npm run version-check && npm run karma",
```

Comment:
I renamed the `karma` task to `unittest`. I'm not quite sure they all qualify as "unit tests" but we want these ones to be fast so that's probably fine.